tests/python_tests/common.py

```diff
@@ -71,59 +71,105 @@ def run_llm_pipeline(
+#     hf_results = run_hugging_face(opt_model, hf_tokenizer, prompts, generation_configs)
+#     ov_results = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
 
-    compare_generation_results(prompts, hf_results, ov_results, generation_configs)
+#     compare_generation_results(prompts, hf_results, ov_results, generation_configs)
 
 
 # TODO: remove after Generator property is supported by LLMPipeline / VLMPipeline
-def generate_and_compare_with_reference_text(models_path: Path,
-                                             prompts: List[str],
-                                             reference_texts_per_prompt: List[List[str]],
-                                             generation_configs: List[GenerationConfig],
-                                             scheduler_config: SchedulerConfig):
-    ov_results : List[GenerationResult] = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
-
-    assert len(prompts) == len(reference_texts_per_prompt)
-    assert len(prompts) == len(ov_results)
-
-    for prompt, ref_texts_for_this_prompt, ov_result in zip(prompts, reference_texts_per_prompt, ov_results):
-        print(f"Prompt = {prompt}\nref text = {ref_texts_for_this_prompt}\nOV result = {ov_result.m_generation_ids}")
-
-        assert len(ref_texts_for_this_prompt) == len(ov_result.m_generation_ids)
-        for ref_text, ov_text in zip(ref_texts_for_this_prompt, ov_result.m_generation_ids):
-            assert ref_text == ov_text
-
+# def generate_and_compare_with_reference_text(models_path: Path,
+#                                              prompts: List[str],
+#                                              reference_texts_per_prompt: List[List[str]],
+#                                              generation_configs: List[GenerationConfig],
+#                                              scheduler_config: SchedulerConfig):
+#     ov_results : List[GenerationResult] = run_continuous_batching(models_path, scheduler_config, prompts, generation_configs)
+
+#     assert len(prompts) == len(reference_texts_per_prompt)
+#     assert len(prompts) == len(ov_results)
+
+#     for prompt, ref_texts_for_this_prompt, ov_result in zip(prompts, reference_texts_per_prompt, ov_results):
+#         print(f"Prompt = {prompt}\nref text = {ref_texts_for_this_prompt}\nOV result = {ov_result.m_generation_ids}")
+
+#         assert len(ref_texts_for_this_prompt) == len(ov_result.m_generation_ids)
+#         for ref_text, ov_text in zip(ref_texts_for_this_prompt, ov_result.m_generation_ids):
+#             assert ref_text == ov_text
+
+
+def generate_and_compare(model: Path | str,
+                         prompts : List[str],
+                         generation_config: List[GenerationConfig] | GenerationConfig | dict,
+                         pipeline_type: PipelineType = PipelineType.PAGED_ATTENTION,
```

Comment:
should default be something like "PipelineType.AUTO" ?
it will denote that default depends on actual backend used in `LLMPipeline`

E.g. if we want to switch back to stateful, we don't need to update this code and `generate_and_compare` will automatically reuse `LLMPipeline` default